### DISCUSSION TITLE
Add option for creating PDF from DOCX with "track changes" shown

### DISCUSSION THREAD

**Discussion by StanNest** [tobya/DocTo#219] [Ideas]
It would be good to have a flag that allows to create a PDF from DOCX with the changes highlighted. (This features would be comparable to the `/markup` option that was present in the (old) OfficeToPDF.exe.)

**Comment by StanNest** [comment]
Ok, found it. The required option is `--ExportMarkup wdExportDocumentWithMarkup`. Great!

**Reply by tobya** [reply]
Great thanks for update
